### DISCUSSION TITLE
67. 커넥션 풀의 개념과 사용 이유에 대해 설명해 주세요.

### DISCUSSION THREAD

**Discussion by cutehumanS2** [jewelry-box/cs-interview#69] [General]
커넥션 풀의 개념과 사용 이유에 대해 설명해 주세요.

**Comment by eun-byeol** [comment]
`DBCP`(DataBase Connection Pool)은 `데이터베이스 연결을 관리`하는 메커니즘입니다. 미리 일정 수의 커넥션을 생성하고 필요할 때 `재사용`할 수 있습니다.

데이터베이스 서버와 커넥션을 맺는 `높은 작업 비용`을 줄이기 위해 DBCP를 사용합니다. 스프링 부트 2.0 이후부터 HikariCP를 기본 DBCP로 채택하고 있습니다.
